requirements/py3_common.txt

```diff
@@ -2,3 +2,12 @@
 
 # Used for Hesiod lookups, etc.
 py3dns==3.1.0
+
+# Django extension for static asset pipeline
+django-pipeline==1.6.8
```

Comment:
@umairwaheed, you should remove this line in #1176, since you would be adding it in requirements/common.txt.